templates/rbac.yaml

```diff
@@ -50,12 +50,24 @@ metadata:
     app.kubernetes.io/version: {{ .Chart.AppVersion }}
     app.kubernetes.io/component: {{ include "coder.serviceName" . }}
 rules:
-  - apiGroups: ["", "apps", "networking.k8s.io"] # "" indicates the core API group
-    resources: ["persistentvolumeclaims", "pods", "deployments", "services", "secrets", "pods/exec","pods/log", "events", "networkpolicies", "serviceaccounts"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims", "pods", "services", "secrets", "serviceaccounts", "pods/exec"]
```

Comment:
I highly doubt all these verbs are sensible on "pods/exec"

Reply:
I broke this out into a separate rule. I think the only verb possible here is `create pods/exec`, correct?